src/hostname/hostnamectl.c

```diff
@@ -370,11 +371,9 @@ static int show_status(int argc, char **argv, void *userdata) {
                 _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
                 const char *text = NULL;
 
-                r = sd_bus_call_method(
+                r = bus_call_method(
                                 bus,
-                                "org.freedesktop.hostname1",
-                                "/org/freedesktop/hostname1",
-                                "org.freedesktop.hostname1",
+                                bus_systemd_mgr,
```

Comment:
This should use the mgr object for /org/freedesktop/hostname1. Since that one doesn't exist yet, it needs to be added to bus-locator.h and bus-locator.c